indexing-solr/src/test/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilterTest.java

```diff
@@ -157,6 +157,11 @@ public void testInventory() throws Exception {
 
         assertEquals("2006-04", dateFormat.format(idb.getDateCreated()));
 
+        assertTrue(idb.getOtherSubject().contains("Canada"));
+        assertTrue(idb.getOtherSubject().contains("Explorer"));
+        assertFalse(idb.getOtherSubject().contains("Snowshoes"));
+        assertFalse(idb.getOtherSubject().contains("rules, boxy"));
```

Comment:
Add a check for the size of the otherSubjects, and then you can remove the two false cases. Checking the size should be more full proof in case we edit any values in the test document and forget to update this.